substrate/frame/uniques/src/tests.rs

```diff
@@ -1099,3 +1099,1268 @@ fn clear_collection_metadata_works() {
 		assert_eq!(Balances::reserved_balance(&1), 10);
 	});
 }
+
+mod asset_ops_tests {
+	use super::*;
+	use crate::asset_strategies::*;
+	use frame_support::traits::tokens::asset_ops::{common_strategies::*, *};
+
+	type Collection = asset_ops::Collection<Uniques>;
+	type Item = asset_ops::Item<Uniques>;
+
+	#[test]
+	fn create_collection() {
+		new_test_ext().execute_with(|| {
+			let collection_id = 10;
+
+			let collection_owner = 1;
+			let collection_admin = 2;
+
+			Balances::make_free_balance_be(&collection_owner, 100);
+
+			assert_ok!(Collection::create(Adminable::new(
```

Comment:
Is `WithAdmin` a better name for `Adminable`?

Reply:
Done

Reply:
Also, the order of the parameters in the constructor functions is changed to be consistent with the order of generics (both for `Owner` and `WithAdmin`)